src/main/java/io/cdap/e2e/pages/locators/CdfControlCenterLocators.java

```diff
@@ -0,0 +1,167 @@
+  public static WebElement hamburgerMenu;
+
+  @FindBy(how = How.XPATH, using = "//*[@data-cy='navbar-control-center-link']")
+  public static WebElement controlCenterMenu;
+
+  @FindBy(how = How.XPATH, using = "//*[@data-cy='feature-heading'][//div[contains(text(),'Control Center')]]")
+  public static WebElement pageHeaderControlCenter;
+
+  @FindBy(how = How.XPATH, using = "//*[@id='create-pipeline-link']")
+  public static WebElement createButtonControlCenter;
+
+  @FindBy(how = How.XPATH, using = "//span[@class='entity-type'][//span[contains(text(),'Data Pipeline')]]")
+  public static WebElement dataPipelineControlCenter;
+
+  @FindBy(how = How.XPATH, using = "//div[@class='just-added-entities-list']//button[@class='btn btn-link']" +
+    "//*[@class='icon-svg icon-trash']")
+  public static WebElement deleteIconControlCenter;
+
+  public static By clickOnDeleteButton() {
+    return By.xpath("//button[@class='btn btn-primary'][//button[@data-cy='Delete']]");
+  }
+
+  public static By clickOnTruncateButton() {
+    return By.xpath("//button[@class='btn btn-primary'][//button[@data-cy='Truncate']]");
+  }
+
+  @FindBy(how = How.XPATH, using = "//div[@class='empty-message-container']")
+  public static WebElement pipelineDeletedMessage;
+
+  @FindBy(how = How.XPATH, using = "//input[@class='search-input form-control'][@placeholder='Search']")
+  public static WebElement searchTabControlCenter;
+
+  @FindBy(how = How.XPATH, using = "//div[@class='just-added-entities-list']//button[@class='btn btn-link']" +
+    "//*['icon-svg icon-wrench']")
+  public static WebElement setPreferencesIcon;
+
+  public static WebElement keyInputField() {
+    return SeleniumDriver.getDriver().findElement(By.xpath("//input[@placeholder='key']"));
+  }
+
+  public static WebElement valueInputField() {
+    return SeleniumDriver.getDriver().findElement(By.xpath("//input[@placeholder='value']"));
+  }
+
+  @FindBy(how = How.XPATH, using = "//button[@data-cy='save-prefs-btn']")
+  public static WebElement clickSaveAndCloseButton;
+
+  @FindBy(how = How.XPATH, using = "//div[@class='just-added-entities-list']" +
+    "//*[@class='icon-svg icon-ETLBatch entity-icon']")
+  public static WebElement clickOnDataPipelineTab;
+
+  @FindBy(how = How.XPATH, using = "//i[@class='fa fa-trash']")
+  public static WebElement clickOnDeleteIconSetPreferences;
+
+  @FindBy(how = How.XPATH, using = "//div[@class='entity-card-header dataset']" +
+    "//*[@class='icon-svg icon-datasets entity-icon']")
+  public static WebElement clickOnDatasetEntityIcon;
+
+  @FindBy(how = How.XPATH, using = "//div[@class='header']//*[@class='icon-svg icon-datasets']")
+  public static WebElement clickOnDatasetEntityDetailsPage;
+
+  @FindBy(how = How.XPATH, using = "(//*[@class='icon-svg icon-trash'])[1]")
+  public static WebElement clickOnDatasetDeleteIcon;
+
+  @FindBy(how = How.XPATH, using = "(//*[@class='icon-svg icon-cut'])[1]")
+  public static WebElement clickOnTruncateDatasetEntity;
```

Comment:
remove click from all locators name because click is a action not a element

Reply:
changed